src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+
+    &:last-child {
+      margin: 0;
+    }
+  }
+
+  &--0 &__star:nth-child(-n + 0),
+  &--1 &__star:nth-child(-n + 1),
+  &--2 &__star:nth-child(-n + 2),
+  &--3 &__star:nth-child(-n + 3),
+  &--4 &__star:nth-child(-n + 4),
+  &--5 &__star:nth-child(-n + 5) {
```

Comment:
The `nth-child` pseudo-class is used here to change the background image of the stars based on a modifier class. However, the `&--0 &__star:nth-child(-n + 0)` selector does not select any stars, as `nth-child(-n + 0)` is not a valid selector that can select elements. If the intention is to have a modifier class that results in no stars being highlighted, this line can simply be removed.